content/index.md

```diff
@@ -12,6 +12,8 @@ Apache OpenJPA is a Java persistence project at [The Apache Software Foundation]
  into any Java EE compliant container and many other lightweight
 frameworks, such as Tomcat and Spring. 
 
+The [4.x releases](downloads.html) is an implementation of the Jakarta Persistence API 3.0 specification is the most current production ready release.
+
 The [3.x releases](downloads.html) targetting the [JSR-338 Java Persistence 2.2](http://jcp.org/en/jsr/detail?id=338) is the most current production ready release.
```

Comment:
Is 3.x still considered "the most current production ready release", even with 4.0.0 released?

Reply:
fixed!